openapi/v2/device-sdk.yaml

```diff
@@ -481,6 +481,39 @@ components:
             value:
               description: "A string representation of the reading's value"
               type: string
+    SecretsRequest:
+      allOf:
+        - $ref: '#/components/schemas/BaseRequest'
+      description: Defines the secret data to be stored
+      type: object
+      properties:
+        path:
+          description: Specifies the type or location in the secret to store
```

Comment:
type or location? And also shouldn't be "...of the secret(s) to store"?

Reply:
Yea, should just be location. Will reword. Remember the ADR definition of a secret? ;-) a collection of key value pairs.

Reply:
And per that definition, the endpoint should be `/secret`, not `/secrets`. Reworking and will submit issue to do same for App services.